heron/tmaster/tests/cpp/server/tmaster_unittest.cpp

```diff
@@ -780,7 +780,7 @@ TEST(StMgr, test_runtime_config) {
     EXPECT_EQ(updated_bolt_config[bolt_init_config + ":runtime"], "4");
   }
   std::map<std::string, std::string> updated_config, updated_spout_config, updated_bolt_config;
-  const heron::proto::system::PhysicalPlan* pplan = common.tmaster_->getPhysicalPlan();
+  auto pplan = common.tmaster_->getPhysicalPlan();
```

Comment:
const is removed? I am wondering if they can make the binary more optimized here.

Reply:
`constr` was removed accidentally. But since it's a test, it doesn't actually matter.